charts/rstudio-library/README.md

```diff
@@ -1,18 +1,16 @@
 # rstudio-library
 
-![Version: 0.1.24](https://img.shields.io/badge/Version-0.1.24-informational?style=flat-square) ![Type: library](https://img.shields.io/badge/Type-library-informational?style=flat-square) ![AppVersion: 0.1.24](https://img.shields.io/badge/AppVersion-0.1.24-informational?style=flat-square)
+![Version: 0.1.25](https://img.shields.io/badge/Version-0.1.25-informational?style=flat-square) ![Type: library](https://img.shields.io/badge/Type-library-informational?style=flat-square) ![AppVersion: 0.1.25](https://img.shields.io/badge/AppVersion-0.1.25-informational?style=flat-square)
 
 #### _Helm library helpers for use by Official RStudio charts_
 
-## Disclaimer
+## For Production
 
-> This chart is "beta" quality. It will likely undergo
-> breaking changes without warning as it moves towards stability.
+To ensure a stable production deployment, please:
 
-As a result, please:
 * Ensure you "pin" the version of the Helm chart that you are using. You can do
   this using the `helm dependency` command and the associated "Chart.lock" files
-  or the `--version` flag. IMPORTANT: This protects you from breaking changes
+  or the `--version` flag. **IMPORTANT: This protects you from breaking changes**
 * Before upgrading, to avoid breaking changes, use `helm diff upgrade` to check
   for breaking changes
 * Pay close attention to [`NEWS.md`](./NEWS.md) for updates on breaking
```

Comment:
Nit: Is it necessary to have this language repeated in so many places? Can it be added to a snippet that's pulled in or moved to a higher level to avoid having to update it in multiple places going forward?

Reply:
Haha yes!! We actually do that 😄 If you notice the [charts/_templates.gotmpl](https://github.com/rstudio/helm/pull/339/files#diff-776f5ec4b6b88b8d9b2281357d65f5d8677508c334e70333182487cac0abe3ab) file, it is the "parent" of these children READMEs. `just docs` (after `just setup`) generates them, and we have CI that makes sure they are up-to-date a well. We still have to commit the files so that they can be viewed (so PRs still look gross) - it'd be nice if we could mark files as "generated" so they are lower priority in the PR review 😞

Reply:
Oh great, that's good to know!